espnet2/tasks/tts.py

```diff
@@ -121,6 +121,21 @@ def add_task_arguments(cls, parser: argparse.ArgumentParser):
             type=str_or_none,
             help="non_linguistic_symbols file path",
         )
+        parser.add_argument(
+            "--cleaner",
+            type=str_or_none,
+            choices=[None, "tacotron", "jaconv", "vietnamese"],
+            default=None,
+            help="Apply text cleaning",
+        )
```

Comment:
Currently, support only a single cleaner, but the use of multiple cleaners can be, right?

Reply:
Exactly, yes. You are great...
Multiple cleaners are also expressed as a class, so I'm not sure which is better.
It's better to consider when such additional cleaner is required really.